modin/core/execution/ray/implementations/pandas_on_ray/io/io.py

```diff
@@ -277,6 +285,13 @@ def to_parquet(cls, qc, **kwargs):
         if not cls._to_parquet_check_support(kwargs):
             return RayIO.to_parquet(qc, **kwargs)
 
+        if len(ray.nodes()) > 1:
+            path = kwargs["path_or_buf"]
+            if is_local_path(path):
+                raise ValueError(
+                    "`path_or_buf` must point to a networked file or buffer when in cluster mode."
+                )
```

Comment:
This can be more succinct:

```suggestion
        if len(ray.nodes()) > 1 and is_local_path(kwargs["path_or_buf"]):
            raise ValueError(
                "`path_or_buf` must point to a networked file or buffer when in cluster mode."
            )        
```